code/go/0chain.net/blobber/worker.go

```diff
@@ -38,8 +37,22 @@ func keepAliveOnChain() {
 
 	for {
 		time.Sleep(REPEAT_DELAY * time.Second)
-		if err := healthCheckOnChain(); err != nil {
+		err := handler.HealthCheckOnChain()
+		handler.SetBlobberHealthError(err)
+		if err != nil {
+			continue // pass // required by linting
```

Comment:
it is better to refactor `healthCheckOnChain`.
```
func healthCheckOnChain() {
	txnHash, err := handler.BlobberHealthCheck(common.GetRootContext())
	if err != nil {
		if err == handler.ErrBlobberHasRemoved {
			return nil
		} else {
			return err
		}
	}

	if t, err := handler.TransactionVerify(txnHash); err != nil {
		logging.Logger.Error("Failed to verify blobber health check", zap.Any("err", err), zap.String("txn.Hash", txnHash))
	} else {
		logging.Logger.Info("Verified blobber health check", zap.String("txn_hash", t.Hash), zap.Any("txn_output", t.TransactionOutput))
	}

	blobberHealthCheckErr = err
}

```

Reply:
Need `blobberHealthCheckErr` variable in handler package (otherwise we'll get circular dependency), But I can move healthCheckOnChain() to worker.go like you're suggesting.